packages/special-pages/index.mjs

```diff
@@ -32,6 +32,11 @@ export const support = {
         'windows': ['copy', 'build-js'],
         'apple': ['copy', 'build-js', 'inline-html'],
     },
+    /** @type {Partial<Record<ImportMeta['injectName'], string[]>>} */
+    errorpage: {
+        'integration': ['copy'],
+        'apple': ['copy', 'inline-html'],
+    },
```

Comment:
@mallexxx I removed the windows output for now - mostly because the CSS styling you added has a bit of macOS specific stuff in there.